pkg/resources/function.go

```diff
@@ -61,9 +66,11 @@ var functionSchema = map[string]*schema.Schema{
 		Type:        schema.TypeString,
 		Description: "The return type of the function",
 		// Suppress the diff shown if the values are equal when both compared in lower case.
-		DiffSuppressFunc: DiffTypes,
-		Required:         true,
-		ForceNew:         true,
+		DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
```

Comment:
`DiffTypes` was doing almost the same but in a case-insensitive manner. Why we are changing that? If this is needed, it should be added to the migration guide as a behavior change.

Reply:
i mean these are both case insensitive, so there is no behavior change. we should probably move this logic to a helper function, but since we also do it this same way in many other places, I took out DiffTypes which is a rather poor name anyways.